fabfile.py

```diff
@@ -92,6 +95,18 @@ def deploy_cold():
     set_symlinks()
 
 
+def build_ui():
+    with lcd('ui'):
+        local('pwd')
+        local('npm install')
+        local('npm run-script bundle')
+
+    with lcd('front_end'):
+        local('npm install')
+        local('gulp build:dist')
```

Comment:
This requires a global install of `gulp`. Wouldn't it be better if we just installed it locally and called this command in the script?